content/tutorials/manage-the-rippled-server/install-rippled.md

```diff
@@ -46,59 +81,193 @@ This section assumes that you are using CentOS 7 or Red Hat Enterprise Linux 7.
+
+        export BOOST_ROOT=/usr/local/Cellar/boost/1.67.0_1
+
+{{n.next()}}. Source the file you added the `BOOST_ROOT` environment variable to. For example:
+
+        source .bashrc
+
+{{n.next()}}. Clone the `rippled` source code into your desired location and access the `rippled` directory.  ***TODO: Note from PM: Cloning github repo is going to require proper github setup. Should we provide some guidance or references? JHA: What is the proper github setup? I may have set this up a whlie ago and I can't remember the steps that may have been required.***
+
+        git clone git@github.com:ripple/rippled.git
+        cd rippled
+
+{{n.next()}}. By default, cloning puts you on the `develop` branch. Use this branch if you are doing development work and want the latest set of untested features.
+
+      If you want the latest stable release, checkout the `master` branch.
+
+        git checkout master
+
+      Or, you can checkout one of the tagged releases listed on [GitHub](https://github.com/ripple/rippled/releases).
+
+      If you want to test out the latest release candidate, checkout the `release` branch:
+
+        git checkout release
+
+{{n.next()}}. Create your `rippled` build directory and access it. For example:
+
+        mkdir my_build
+        cd my_build
+
+{{n.next()}}. Build `rippled`. This could take about 5 minutes, depending on your hardware specs.
+
+        cmake -G "Unix Makefiles" -DCMAKE_BUILD_TYPE=Debug ..
+
+      You can set `CMAKE_BUILD_TYPE` to the `Debug` or `Release` build type. All four standard [`CMAKE_BUILD_TYPE`](https://cmake.org/cmake/help/v3.0/variable/CMAKE_BUILD_TYPE.html) values are supported.
+
+{{n.next()}}. Run the build using CMake. This could take about 10 minutes, depending on your hardware specs.
+
+        cmake --build . -- -j 4
+
+      This example uses a `-j` parameter set to `4`, which uses four processes to build in parallel. The best number of processes to use depends on how many CPU cores your hardware has available. Use `sysctl -a | grep machdep.cpu | grep core_count` to get your CPU core count. ***TODO: the mac build page in the github repo goes much more in-depth about cmake build options - should we include these here? https://github.com/ripple/rippled/tree/develop/Builds/macos#generate-and-build***
```

Comment:
I don't think its necessary and did not use it myself. Not sure what the background for those instructions are however, maybe Rome knows more?

Reply:
Sounds great - let's see if Manoj and Rome have feedback. I'm going to add Rome as a reviewer once I get a first round of reviews incorporated. 👍

Reply:
When I updated the Ubuntu equivalent of these instructions, I didn't want to overwhelm the reader with options and possibilities, so I left a couple of tips but didn't go into extreme detail. I also don't like relying on a series of several piped commands including a `grep` for getting your CPU count, since a minor change in the OS's output format could break that. I went with a slightly more generic message about where to get information about your processor.

These instructions can probably be based on the changes I made there: https://github.com/ripple/ripple-dev-portal/blame/master/content/tutorials/manage-the-rippled-server/build-run-rippled-ubuntu.md#L107

Reply:
Great - I will leave out the reference to the in-depth cmake build options. 👍 

@mDuo13 - Regarding the grep - I added the command because `cat /proc/cpuinfo` didn't work for me in macOS. I see your point about it being brittle! Is there a more stable way to get this info for macOS? Or perhaps I should just leave the command out for macOS?

Reply:
YES! 🙌 Thank you!